installer/client/src/components/installer-layout.scss

```diff
@@ -0,0 +1,52 @@
+// OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+// all the essential functionalities required for any enterprise.
+// Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+//
+// OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+// the GNU General Public License as published by the Free Software Foundation; either
+// version 2 of the License, or (at your option) any later version.
+//
+// OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+// without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+// See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with this program;
+// if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301, USA
+
+.orangehrm-installer {
+  height: inherit;
+  display: grid;
+  grid-template:
+    "sidebar header"
+    "sidebar content" / minmax(25%, 240px) auto;
+  grid-template-rows: 120px 1fr;
+  &-header {
+    grid-area: header;
+    margin-left: auto;
+    padding: 1rem 2rem;
+    box-sizing: border-box;
+  }
+  &-sidebar {
+    position: relative;
+    grid-area: sidebar;
+    padding: 0 2rem;
+    box-sizing: border-box;
+    background: linear-gradient(180deg, #fa9e2e 0%, #e55422 90%);
+    // background: linear-gradient(
+    //   180deg,
+    //   $oxd-primary-gradient-start-color 0%,
+    //   $oxd-primary-gradient-end-color 90%
+    // );
```

Comment:
*

Reply:
Have some concerns using colors for gradient, the colors in design and the colors in oxd guide don't exactly match. so i kept the correct styles untill we can finalize.